src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/IncidentMappingResourceIntTest.kt

```diff
@@ -134,21 +134,21 @@ class IncidentMappingResourceIntTest : IntegrationTestBase() {
         .returnResult().responseBody
 
       with(responseBody!!) {
-        assertThat(userMessage).contains("Conflict: Incident mapping already exists.\nExisting mapping: IncidentMappingDto(nomisIncidentId=1234, incidentId=4444, label=2022-01-01, mappingType=NOMIS_CREATED")
-        assertThat(userMessage).contains("Duplicate mapping: IncidentMappingDto(nomisIncidentId=21, incidentId=4444, label=2022-01-01, mappingType=NOMIS_CREATED, whenCreated=null)")
+        assertThat(userMessage).contains("Conflict: Incident mapping already exists.\nExisting mapping: IncidentMappingDto(nomisIncidentId=1234, incidentId=4321, label=2022-01-01, mappingType=NOMIS_CREATED")
+        assertThat(userMessage).contains("Duplicate mapping: IncidentMappingDto(nomisIncidentId=21, incidentId=4321, label=2022-01-01, mappingType=NOMIS_CREATED, whenCreated=null)")
```

Comment:
very finickity (and other syncs have this too) but i suppose these '4321's should be $INCIDENT_ID

Reply:
yes - I did a full pull out of these to make it more identifiable - will update in next PR